src/vmm/src/arch/aarch64/pvtime.rs

```diff
@@ -18,7 +18,7 @@ pub const STEALTIME_STRUCT_MEM_SIZE: u64 = 64;
 #[derive(Debug)]
 pub struct PVTime {
     /// Number of vCPUs
-    vcpu_count: u8,
+    vcpu_count: u64,
```

Comment:
Please move rust related fixes/changes into the first one, so they are not scattered. Let second commit be just a python integration test.